packages/core/src/rpc-client.ts

```diff
@@ -157,7 +158,10 @@ executeRpcCall.warm = (apiUrl: string) => {
 }
 
 const getApiUrlFromResolverFilePath = (resolverFilePath: string) =>
-  resolverFilePath.replace(/^app\/_resolvers/, "/api")
+  resolverFilePath.replace(
+    /^app\/_resolvers/,
+    "/api".concat(getBlitzRuntimeData().trailingSlash ? "/" : ""),
+  )
```

Comment:
You need to contact the slash on end of resolverFilePath, not `api`. Current code will result in url like `/api//queries/getUser`